pvlib/ivtools/sde.py

```diff
@@ -124,8 +125,8 @@ def fit_sandia_simple(voltage, current, v_oc=None, i_sc=None, v_mp_i_mp=None,
     .. math::
 
         \log(\beta_{0} - \beta_{1} V - I)
-        &\approx \log(\frac{I_{0}}{1 + G_{p} R_{s}} + \frac{V}{nN_sV_{th}}
-        + \frac{I R_{s}}{nN_sV_{th}}) \\
+        &\approx \log(\frac{I_{0}}{1 + G_{p} R_{s}}) + \frac{V}{nN_sV_{th}}
+        + \frac{I R_{s}}{nN_sV_{th}} \\
```

Comment:
@cwhanse can you confirm that this change is correct?

Reply:
It is, thanks.